packages/web/config/price.ts

```diff
@@ -553,4 +553,17 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     spotPriceDestDenom: "uosmo",
     destCoinId: "osmosis",
   },
+  {
+    alternativeCoinId: "pool:lambda",
+    poolId: "823",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-378" }],
```

Comment:
In the chain registry, it says that channel 350 is the channel id, not 378. Are you able to reconsider and confirm which is the correct channel-id, please?
https://github.com/cosmos/chain-registry/blob/master/_IBC/lambda-osmosis.json

Reply:
nvm, I saw the PR that corrects the info in the chain registry

Reply:
We have replaced expired client and set new channel